core/src/main/java/org/apache/servicecomb/core/definition/SchemaMeta.java

```diff
@@ -83,40 +83,42 @@ public String getPackageName() {
   }
 
   private void initOperations() {
-    for (Entry<String, Path> entry : swagger.getPaths().entrySet()) {
-      String strPath = entry.getKey();
-      Path path = entry.getValue();
-      for (Entry<HttpMethod, Operation> operationEntry : path.getOperationMap().entrySet()) {
-        Operation operation = operationEntry.getValue();
-        if (operation.getOperationId() == null) {
-          throw ExceptionUtils.operationIdInvalid(getSchemaId(), strPath);
+    if (swagger.getPaths() != null) {
```

Comment:
I think we to write a logger for it, so use can know what's wrong here.

Reply:
I think no necessary, user want fill this empty interface later, so it's not a `wrong scenario`

Reply:
emmm, ok , add an error output in order to notice user like
`swagger definition for org.apache.servicecomb.samples.uselog4j2.HelloImpl with path /hello is an empty interface, please delete it or fill with one method!`